app/ntpd/README.md

```diff
@@ -24,74 +12,60 @@ that a local cluster of ntpd services stays in sync with an NTP pool.
 Basics that do not differ from the client to the server are in a common
 template which the individual server and client templates reference.
 
-### Items
-
-#### Common
-
-* Memory usage (rss) of "ntpd" processes (proc.mem[ntpd,ntp,,,rss])
-* Number of "ntpd" processes (proc.num[ntpd,ntp])
-* ntpd system peer (rabe.ntpdc.sysinfo[system peer])
-* ntpd system peer mode (rabe.ntpdc.sysinfo[system peer mode])
-* ntpd stratum (rabe.ntpdc.sysinfo[stratum])
-* ntpd precision (rabe.ntpdc.sysinfo[precision])
-* ntpd root distance (rabe.ntpdc.sysinfo[root distance])
-* ntpd root dispersion (rabe.ntpdc.sysinfo[root dispersion])
-* ntpd system flags (rabe.ntpdc.sysinfo[system flags])
-* ntpd jitter (rabe.ntpdc.sysinfo[jitter])
-* ntpd stability (rabe.ntpdc.sysinfo[stability])
-* ntpd broadcastdelay (rabe.ntpdc.sysinfo[broadcastdelay])
-* ntpd authdelay (rabe.ntpdc.sysinfo[authdelay])
-
-#### Client
-
-The following Items are created for each discovered Server
-
-* NTP server {#SERVERNAME} candidate order (rabe.ntpd.server.candidate_order[{#SERVERNAME}])
-* NTP server {#SERVERNAME} configuration (vfs.file.regexp[/etc/ntp.conf,"^server.*{#SERVERNAME} (.*)",,,,\1])
+This template is part of [RaBe's Zabbix template and helpers
+collection](https://github.com/radiorabe/rabe-zabbix).
 
-#### Server
+## Template App ntpd Client active
+### Discovery
+
+#### Discovery Items
+* NTP server {#SERVERNAME} candidate order (`rabe.ntpd.server.candidate_order[{#SERVERNAME}]`)
+* NTP server {#SERVERNAME} configuration (`vfs.file.regexp[/etc/ntp.conf,"^server.*{#SERVERNAME} (.*)",,,,\1]`)
+#### Discovery Triggers
+* NTP server {#SERVERNAME} is not a valid candidate on {HOST.NAME} (`{Template App ntpd Client active:rabe.ntpd.server.candidate_order[{#SERVERNAME}].last()}=0`)
+
+If a configured server has a candidate order of 0 it is not considered as a valid time source and most likely has some issues.
+
+## Template App ntpd Common active
+
+### Items 
+* Memory usage ($5) of "$1" processes (`proc.mem[ntpd,ntp,,,rss]`)
+* Number of "$1" processes (`proc.num[ntpd,ntp]`)
+* ntpd $1 (`rabe.ntpdc.sysinfo[authdelay,single]`)
```

Comment:
Yes, missing since it's not in the template. https://github.com/radiorabe/rabe-zabbix/blob/14c2b96e71722c1c9321154803007ca3c05a1ff7/app/zabbix-agent/README.md has an example of items with a description in the template.